src/api/routes/index.js

```diff
@@ -1,9 +1,13 @@
 import express from 'express';
 import authRouter from './authRouter';
 import resetRouter from './password.reset';
+import article from './articlesRouter';
 
 const router = express();
 
+router.use('/auth', authRouter);
+router.use('/reset', resetRouter);
```

Comment:
@truestbyheart please ensure you're up-to-date with what's on develop. You shouldn't have code related to `auth` or `reset` in your work.